powerline.el

```diff
@@ -506,8 +506,21 @@ static char * %s[] = {
     (setq powerline-selected-window (frame-selected-window))))
 
 (add-hook 'window-configuration-change-hook 'powerline-set-selected-window)
+
+;; focus-in-hook was introduced in emacs v24.4.
+;; Gets evaluated in the last frame's environment.
 (add-hook 'focus-in-hook 'powerline-set-selected-window)
-(add-hook 'focus-out-hook 'powerline-set-selected-window)
+
+;; focus-out-hook was introduced in emacs v24.4.
+(add-hook 'focus-out-hook (lambda ()
```

Comment:
could you extract the lambda to a separate function. maybe `powerline-unset-selected-window`. This will make it easier if we need to remove the hook.